lesson17/main.js

```diff
@@ -0,0 +1,123 @@
+const jsonUrl = "https://jsondata.okiba.me/v1/json/YbIZe211030061143";
+const wrapper = document.getElementById("js-slideshow");
```

Comment:
```suggestion
const slideshowWrap = document.getElementById("js-slideshow");
```

Reply:
Thank you. I changed its name to make more clearly.
[4a6d393](https://github.com/yuka830/js-lessons/pull/21/commits/4a6d39375a7bbceeef964514784e59f813ba5cb1)